services/api/routers/minio.py

```diff
@@ -120,8 +123,36 @@ async def post_discover_upload(file: UploadFile):
             raise e
     else:
         return { 'object_name': stat.object_name, 'etag': stat.etag }
-    # upload
+
+    # upload original size image
     upload = storage.put_object(crd.minio.bucket, object_name, file.file, length=-1, part_size=10*1024*1024)
+
+    # if file is an image, create and upload thumbnail image
+    supported_formats = {
```

Comment:
Should this be extracted to a config file?

Reply:
Yes, please :) Extract it to `api.config`.

Reply:
And please add `'image/webp': 'webp',`.